<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mod</class>
 <widget class="QDialog" name="Mod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改信息</string>
  </property>
  <layout class="QVBoxLayout" name="vlay" stretch="2,1,2">
   <property name="leftMargin">
    <number>48</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>48</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label1">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>修改信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="flay">
     <property name="horizontalSpacing">
      <number>36</number>
     </property>
     <property name="verticalSpacing">
      <number>8</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label2">
       <property name="text">
        <string>新密码</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ledit1"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label3">
       <property name="text">
        <string>新名称</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label4">
       <property name="text">
        <string>新电话</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ledit2"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="ledit3"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label5">
       <property name="text">
        <string>新位置</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="ledit4"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlay2">
     <item alignment="Qt::AlignmentFlag::AlignVCenter">
      <widget class="QPushButton" name="pbtn1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignVCenter">
      <widget class="QPushButton" name="pbtn2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>注销</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignVCenter">
      <widget class="QPushButton" name="pbtn3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
